cmd/docker-app/merge.go

```diff
@@ -1,29 +1,62 @@
 package main
 
 import (
+	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
+	"strings"
 
-	"github.com/docker/app/internal"
 	"github.com/docker/app/internal/packager"
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
 var mergeOutputFile string
 
+// Check appname directory for extra files and return them
+func extraFiles(appname string) ([]string, error) {
+	files, err := ioutil.ReadDir(appname)
+	if err != nil {
+		return nil, err
+	}
+	var res []string
+	for _, f := range files {
+		switch f.Name() {
+		case "docker-compose.yml":
+		case "metadata.yml":
+		case "settings.yml":
```

Comment:
I think this now should use `Internal.ComposeFile` etc…
But maybe if we want to generalize a bit we should loop on `internal.FileNames`?